crates/executor/src/util.rs

```
@@ -29,8 +29,8 @@ pub(crate) fn receipt_envelope_from_parts<'a>(
         OpTxType::Eip1559 => {
             OpReceiptEnvelope::Eip1559(ReceiptWithBloom { receipt: inner_receipt, logs_bloom })
         }
-        OpTxType::Eip4844 => {
-            OpReceiptEnvelope::Eip4844(ReceiptWithBloom { receipt: inner_receipt, logs_bloom })
+        OpTxType::Eip7702 => {
+            OpReceiptEnvelope::Eip7702(ReceiptWithBloom { receipt: inner_receipt, logs_bloom })
```

We cannot use 7702, Pectra isn't active - this should throw an error.